src/App.scss

```diff
@@ -15,25 +15,50 @@ button {
   color: red;
 }
 
+.App {
+  height: 100vh;
+  padding-top: 15px;
+  background: rgb(36,47,61);
+
+  &__TodoForm {
+    background-color: rgb(17, 27, 41);
+    margin: 0 auto;
+    width: 350px;
+    padding: 10px;
+    border: 1px;
```

Comment:
Consider adding a unit to the border property. For example, `border: 1px solid black;`. 📏